helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -67,74 +65,64 @@ starrocksFESpec:
     loadbalancerIP: ""
     # add annotations for fe service.
     annotations: {}
-
-  # contains the secret name
+  # imagePullSecrets allows you to use secrets to pull images for pods.
   imagePullSecrets: []
-  # - name: "image-pull-secret"
-
+    # - name: "image-pull-secret"
   # serviceAccount for fe access cloud service.
   serviceAccount: ""
-
   # If specified, the pod's nodeSelector，displayName="Map of nodeSelectors to match when scheduling pods on nodes"
   # Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
   nodeSelector: {}
-  #  beta.kubernetes.io/arch: amd64
-  #  beta.kubernetes.io/os: linux
-
+    # beta.kubernetes.io/arch: amd64
+    # beta.kubernetes.io/os: linux
   # the pod labels for user select or classify pods.
   podLabels: {}
-
-  ## hostAliases allows adding entries to /etc/hosts inside the containers
+  # hostAliases allows adding entries to /etc/hosts inside the containers
   hostAliases: []
-  # - ip: "127.0.0.1"
-  #   hostnames:
-  #   - "example.com"
-
+    # - ip: "127.0.0.1"
+    #   hostnames:
+    #   - "example.com"
   # schedulerName allows you to specify which scheduler will be used for your pods.
   schedulerName: ""
-
   # Additional fe container environment variables
   # You specify this manually like you would a raw deployment manifest.
   # This means you can bind in environment variables from secrets.
   # Ref: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
   feEnvVars: []
-  # e.g. static environment variable:
-  #  - name: DEMO_GREETING
-  #    value: "Hello from the environment"
-  #
-  # e.g. secret environment variable:
-  #  - name: USERNAME
-  #    valueFrom:
-  #      secretKeyRef:
-  #        name: mysecret
-  #        key: username
-
-  # Pod affinity
+    # e.g. static environment variable:
```

Comment:
we can add a kubernetes doc link here, here only lists two types of env var, but kubernetes supports more than that, we can lead the user to check the doc on how to set the env var themselves.

Reply:
good suggestion, I will fix it.